LunaMary/src/AdditionalClasses/OurStringArrayList.java

```diff
@@ -0,0 +1,66 @@
+package AdditionalClasses;
+
+public class OurStringArrayList {
+    String[] list;
+    Integer length;
+    Integer position;
+
+    public OurStringArrayList(Integer length) {
+        this.length = length;
+        this.list = new String[length];
+        this.position = 0;
+
+    }
+
+    public void add(String text) {
+        list[position] = text;
+        position++;
+    }
+
+    public void remove(Integer position) {
```

Comment:
This method needs to update the position of the class, something like:
this.position--;